miden-lib/asm/kernels/transaction/api.masm

```diff
@@ -71,72 +72,76 @@ end
 
 #! Returns the account id.
 #!
-#! Stack: [0]
-#! Output: [acct_id]
+#! Inputs:  [pad(16)]
+#! Outputs: [acct_id, pad(15)]
 #!
 #! Where:
 #! - acct_id is the account id.
 export.get_account_id
     # get the account id
     exec.account::get_id
-    # => [acct_id, 0]
+    # => [acct_id, pad(16)]
 
+    # truncate the stack
     swap drop
-    # => [acct_id]
+    # => [acct_id, pad(15)]
 end
 
 #! Returns the account nonce.
 #!
-#! Stack: [0]
-#! Output: [nonce]
+#! Inputs:  [pad(16)]
+#! Outputs: [nonce, pad(15)]
 #!
 #! Where:
 #! - nonce is the account nonce.
 export.get_account_nonce
     # get the account nonce
     exec.account::get_nonce
-    # => [0, nonce]
+    # => [nonce, pad(16)]
 
+    # truncate the stack
     swap drop
-    # => [nonce]
+    # => [nonce, pad(15)]
 end
 
 #! Returns the initial account hash.
 #!
-#! Stack: [0, 0, 0, 0]
-#! Output: [H]
+#! Inputs:  [pad(16)]
+#! Outputs: [H, pad(12)]
 #!
 #! Where:
 #! - H is the initial account hash.
 export.get_initial_account_hash
```

Comment:
I totally agree! My initial though was to move the update of the names to another PR (since this one is already huge, and I wanted to leave this one to be a format+ABI only (but I already failed to do so :) )), but probably it is not a common problem, so I can update the names here.